bqetl_project.yaml

```diff
@@ -408,6 +408,12 @@ generate:
       firefox_desktop:
         # firefox_desktop is a single app containing multiple channels hence why set to null.
         channels: null
+      focus_android:
+        channels:
+        - release
```

Comment:
I see `org_mozilla_focus_nightly` and `org_mozilla_focus_beta`. I can't imagine that there are any substantial amount of users on these channels, but what would be the additional work needed to include them?

Reply:
Added those.